test/unittests/commands/excel2json/new_lists/test_compliance_checks.py

```diff
@@ -87,11 +87,12 @@ def test_content_compliance(self, f1_s1_good_en: ExcelSheet, f2_s2_missing_trans
             "The excel sheet 'sheet2' has the following problem(s):\n"
             "In one list, all the nodes must be translated into all the languages used. "
             "For the following nodes, the translations are missing:\n"
+            "    - Row Number: 2 | Column(s): de_list\n"
             "    - Row Number: 3 | Column(s): en_1\n"
             "    - Row Number: 8 | Column(s): de_1"
         )
         with pytest.raises(InputError, match=expected):
-            _check_for_missing_translations_all_excels(all_sheets)
+            make_all_excel_compliance_checks(all_sheets)
```

Comment:
I noticed that I erroneously called the function that does this check directly here. Although I do that elsewhere too.